.circleci/config.yml

```diff
@@ -479,6 +490,34 @@ jobs:
       - store_artifacts:
           path: /tmp/screenshots
 
+  test-nightly:
+    <<: *job-defaults-macos
+    working_directory: /Users/distiller/project/ios
+    environment:
+      <<: *env-macos
+    steps:
+      - <<: *checkout-macos
+      - run:
```

Comment:
Maybe extract this and below to an anchor now as it will be needed in each release job anyways? I found it easy to make typos when doing this type of refactor, it would also reduce the changes needed to use in release jobs.